### DISCUSSION TITLE
Distortion with animated water drops on Rain v2.

### DISCUSSION THREAD

**Discussion by herboren** [rocksdanister/lively#743] [General]
**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.
![rZfxiePq6q](https://user-images.githubusercontent.com/987794/134249585-fdf1654e-903d-4234-81f5-0eda65421c31.gif)

**Describe the bug**
Every period of time > 12 hrs the animated of falling water droplets start showing distortion.

**To Reproduce**
Leaving it running more than 12 hours

**Expected behavior**
Smooth animation

**Desktop (please complete the following information):**
Microsoft Windows [Version 10.0.18363.1801]
Rain v2

**Additional context**
Can be fixed if wallpaper is re-applied. Select a different wallpaper then back to Rain v2, redrawing of the screen rids the distortion.

**Log file (Important)**
Logged attached to discussion reply.

**Comment by rocksdanister** [comment]
I moved this to discussion since this is a bug on the wallpaper side and not Lively application itself.

**Reply by herboren** [reply]
Ah OK, rock on.
Here is the log if needed still.
[lively_log_20210921_173129.zip](https://github.com/rocksdanister/lively/files/7206423/lively_log_20210921_173129.zip)

**Reply by rocksdanister** [reply]
Its likely because one of the variable is overflowing in the raindrop shader code.. log won't help since its not a problem on Lively's side.
